app/assets/javascripts/application.js

```diff
@@ -13,4 +13,73 @@
 //= require jquery
 //= require jquery_ujs
 //= require turbolinks
+//= require bootstrap
+//= require interact/interact
 //= require_tree .
+
+interact('.resize-drag')
+    .draggable({
+        onmove: window.dragMoveListener
+    })
+    .resizable({
```

Comment:
this is difficult to use UX-wise right now. I would take another look at resizing